src/Preflight/Preflight.php

```diff
@@ -313,6 +334,7 @@ public function preflight($argv): bool
         // redispatch to a site-local Drush, then we cannot continue.
         // This can happen when using Drush 9 to call a site-local Drush 8
         // using an alias record that is only defined in a Drush 8 format.
+        // TODO: We can remove this when we drop support for Drush 8 aliases (already done?)
```

Comment:
yes, already done afaik

Reply:
This comment appears to be incorrect; we get here any time the user tries to use an alias that cannot be found.